exchanges/btse/btse_wrapper.go

```diff
@@ -350,7 +352,23 @@ func (b *BTSE) UpdateTicker(ctx context.Context, p currency.Pair, a asset.Item)
 	if !b.SupportsAsset(a) {
 		return nil, fmt.Errorf("%w %v", asset.ErrNotSupported, a)
 	}
-	if err := b.UpdateTickers(ctx, a); err != nil {
+	ticks, err := b.GetMarketSummary(ctx, p.String(), a == asset.Spot)
+	if len(ticks) != 1 {
+		return nil, errors.New("market_summary should return 1 tick for a single ticker")
+	}
+	if err == nil {
+		err = ticker.ProcessTicker(&ticker.Price{
+			Pair:         p,
+			Ask:          ticks[0].LowestAsk,
+			Bid:          ticks[0].HighestBid,
+			Low:          ticks[0].Low24Hr,
+			Last:         ticks[0].Last,
+			Volume:       ticks[0].Volume,
+			High:         ticks[0].High24Hr,
+			ExchangeName: b.Name,
+			AssetType:    a})
+	}
+	if err != nil {
```

Comment:
I don't understand why checking the length of the response is a higher priority than checking the returned error,  dumping whatever BTSE has said about the situation.

Reply:
Yeah. That's just wrong.
Suspect it got abstracted several times and then ended up like this.

Fixed